test/GithubApi.Issue.test.js

```diff
@@ -0,0 +1,73 @@
+const agent = require('superagent');
+const { expect } = require('chai');
+
+const urlApi = 'https://api.github.com';
+const namePublicRepo = 'workshop-api-testing-js';
+
+describe('Given a github user', () => {
+  let user;
+  let repository;
+
+  before(async () => {
+    const response = await agent.get(`${urlApi}/user`)
+      .auth('token', process.env.ACCESS_TOKEN)
+      .set('User-Agent', 'agent');
+
+    user = response.body;
+    const listResponse = await agent.get(user.repos_url)
+      .auth('token', process.env.ACCESS_TOKEN)
+      .set('User-Agent', 'agent');
+
+    repository = listResponse.body.find((list) => list.name === namePublicRepo);
+  });
+
+  describe('when wanna all repositories', () => {
+    it('Then should have public repositories', () => {
+      expect(user.public_repos).to.be.greaterThan(0);
+    });
+  });
+
+  describe('And wanna select repository', () => {
+    it('Then should have the public repository', () => {
+      expect(repository).to.not.equal(undefined);
+    });
+  });
+  describe('And wanna create a new issue', () => {
```

Comment:
I would propose a different level for this And, since you are creating an Issue for the *SELECTED* repository in the step before